src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 1)).toStrictEqual([8]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(6, 2)).toStrictEqual([3, 3]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(17, 4)).toStrictEqual([4, 4, 4, 5]);
 });
 
 test('should add zeros if value < numberOfParts', () => {
-
+  expect(splitInteger(55, 10)).toStrictEqual([5, 5, 5, 5, 5, 6, 6, 6, 6, 6]);
```

Comment:
Please, recheck the conditions in this test case. Your test does not check to add zero.

Reply:
Done